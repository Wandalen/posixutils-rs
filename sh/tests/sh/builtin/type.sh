alias a='echo hello'
f () {
  :
}

type a
type f
type set exit 
type -- type
