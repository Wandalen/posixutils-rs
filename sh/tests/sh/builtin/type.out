a is aliased to 'echo hello'
f is a function
set is a special shell builtin
exit is a special shell builtin
type is a shell builtin
